bin/generate_charts.py

```diff
@@ -381,52 +370,53 @@ def generate_charts(files):
                         required=False,
                         type=str,
                         help='Location where all output will be stored')
+    parser.add_argument('--coalesce-workloads',
+                        dest='coalesce',
+                        action='store_true',
+                        help='Specify put all workloads on a single set of charts')
+
     args = parser.parse_args()
 
     prefixes = {}
 
     if args.output != '':
         output = path.join(args.output, '')
 
+    coalesce = args.coalesce
 
     # Recursively fetch all json files in the results dir.
-    for file in glob.iglob(path.join(args.results_dir, "**/*.json"), recursive=True):
-        prefix = path.dirname(file)
-        if prefix in prefixes:
-            prefixes[prefix].append(file)
-        else:
-            prefixes[prefix] = [file]
+    filelist = glob.iglob(path.join(args.results_dir, "**/*.json"), recursive=True)
 
-    for prefix in prefixes:
-        generate_charts(prefixes[prefix])
+    generate_charts(filelist)
 
     html = '''
 <html>
 <head>
-
+<script>
```

Comment:
I'm a JS noob, so hoping this works ok.

Reply:
Taken from here: https://github.com/Kozea/pygal.js/blob/gh-pages/2.0.x/pygal-tooltips.min.js
Unfortunately you can't link out otherwise it breaks CORS rules